solana/bridge-escrow/programs/bridge-escrow/src/tests.rs

```diff
@@ -397,82 +385,128 @@ fn escrow_bridge_program() -> Result<()> {
         amount_out
     );
 
-    // If above fails -> timeout
+    // Store the intent
+    println!("\nStore the intent for cross domain");
+    let intent_id = "12323543".to_string();
 
-    // // Derive the necessary accounts
-    // let (storage, _bump_storage) = Pubkey::find_program_address(
-    //     &[solana_ibc::SOLANA_IBC_STORAGE_SEED],
-    //     &solana_ibc::ID,
-    // );
-    // let (trie, _bump_trie) =
-    //     Pubkey::find_program_address(&[solana_ibc::TRIE_SEED], &solana_ibc::ID);
-    // let (chain, _bump_chain) = Pubkey::find_program_address(
-    //     &[solana_ibc::CHAIN_SEED],
-    //     &solana_ibc::ID,
-    // );
-    // let (mint_authority, _bump_mint_authority) = Pubkey::find_program_address(
-    //     &[solana_ibc::MINT_ESCROW_SEED],
-    //     &solana_ibc::ID,
-    // );
-    // let (escrow_account, _bump_escrow_account) = Pubkey::find_program_address(
-    //     &[solana_ibc::ESCROW, &hashed_full_denom.as_slice()],
-    //     &solana_ibc::ID,
-    // );
+    // arbitrary value
+    let amount_out = 10000;
 
-    // let receiver_token_account = get_associated_token_address(
-    //     &authority.pubkey(),
-    //     &native_token_mint_key,
-    // );
-    // let (fee_collector, _bump_fee_collector) =
-    //     Pubkey::find_program_address(&[solana_ibc::FEE_SEED], &solana_ibc::ID);
+    let intent_state = Pubkey::find_program_address(
+        &[crate::INTENT_SEED, intent_id.as_bytes()],
+        &crate::ID,
+    )
+    .0;
+
+    let new_intent = IntentPayload {
+        intent_id: intent_id.clone(),
+        user_in: user.pubkey(),
+        token_in,
+        amount_in: TRANSFER_AMOUNT,
+        token_out: token_out.to_string(),
+        amount_out: amount_out.to_string(),
+        timeout_timestamp_in_sec: 10000,
```

Comment:
That’s 3 am on 1 of January 1970. If this is meant to represent relative timeout the field should be called `timeout_in_set`. `timestamp` implies this is seconds since Unix epoch.

Reply:
yes, my bad. It is supposed to be timestamp.